ui/src/table.tsx

```diff
@@ -450,6 +454,10 @@ const
         wave.args[m.name] = [item.key as S]
         wave.push()
       },
+      getLinkComp = (str: S, target?: S) => {
+        const [, label, path = str] = /\[(.+)\].*\((.+)\)/.exec(str) || []
```

Comment:
We do not expect anything between `](` I guess. Would this work?
```suggestion
        const [, label, path = str] = /\[(.+)\]\((.+)\)/.exec(str) || []
```

Reply:
It was part of an old regex I used, not needed anymore. Removed.